mkosi/__init__.py

```diff
@@ -2754,7 +2776,7 @@ def jj(line: str) -> str:
         patch_file(os.path.join(root, "etc/pam.d/common-auth"), jj)
 
 
-def install_distribution(args: CommandLineArguments, root: str, *, do_run_build_script: bool, cached: bool) -> None:
```

Comment:
Is there a reason to relax this? The argument lists are often awfully long and I do hate counting, so I do prefer when keyword arguments are forced to be used with their names.

Reply:
Consistency, all the other functions work the same way and are called without keyword arguments in `build_image`. You have a point that using keyword arguments might be better but let's make sure we do it for all the callsites then instead of for only one.

Reply:
I see, I guess I would have left it in, to change this for the others, but it's fine either way, since there are no external users.